tools/minibrowser/build.gradle

```diff
@@ -44,4 +60,7 @@ dependencies {
     implementation 'com.google.android.material:material:1.7.0'
     implementation 'androidx.core:core-ktx:1.9.0'
     implementation 'androidx.fragment:fragment-ktx:1.5.4'
+    implementation "androidx.navigation:navigation-fragment-ktx:2.5.3"
+    implementation "androidx.navigation:navigation-ui-ktx:2.5.3"
```

Comment:
You should use '' instead of "" for strings to be coherent with the rest of the file (lines 63 and 64)

Reply:
same again :see_no_evil:

Reply:
fixed